fennel/lib/to_proto/serializer.py

```diff
@@ -164,6 +177,22 @@ def visitFilter(self, obj):
             ),
         )
 
+    def visitAssign(self, obj):
+        assign_func_pycode = to_includes_proto(obj.func)
+        gen_pycode = self.wrap_function(
+            assign_func_pycode, is_assign=True, column_name=obj.column
+        )
+
+        return proto.Operator(
+            id=obj.signature(),
+            id_root=(obj == self.terminal_node),
+            pipeline_name=self.pipeline_name,
+            dataset_name=self.dataset_name,
+            assign=proto.Filter(
```

Comment:
why is Assign returning a filter ?

Reply:
Can you add a test to datasets/test_datasets.py for checking if the generated proto makes sense